Formula/gcc.rb

```diff
@@ -87,21 +84,25 @@ def install
       --with-zstd=#{Formula["zstd"].opt_prefix}
       --with-pkgversion=#{pkgversion}
       --with-bugurl=#{tap.issues_url}
+      --with-system-zlib
     ]
-    # libphobos is part of gdc
-    args << "--enable-libphobos" if Hardware::CPU.intel?
 
     if OS.mac?
       cpu = Hardware::CPU.arm? ? "aarch64" : "x86_64"
       args << "--build=#{cpu}-apple-darwin#{OS.kernel_version.major}"
-      args << "--with-system-zlib"
+
+      # libgcc_s.1.dylib links to `@rpath/libgcc_s.1.1.dylib`.
+      ENV.append "LDFLAGS", "-Wl,-rpath,@loader_path"
 
       # System headers may not be in /usr/include
       sdk = MacOS.sdk_path_if_needed
-      args << "--with-sysroot=#{sdk}" if sdk
+      if sdk
+        args << "--with-native-system-header-dir=/usr/include"
```

Comment:
`--with-native-system-header-dir` is not needed anymore, I just forgot to remove it in the `gcc@12`.

Reply:
Oh, oops. We can remove it in a re-run (if we need one).

Reply:
Fixed.